service/common/src/main/java/org/apache/polaris/service/exception/IcebergJsonProcessingExceptionMapper.java

```diff
@@ -76,6 +76,7 @@ public Response toResponse(JsonProcessingException exception) {
      * Otherwise, it's those pesky users.
      */
     LOGGER.info("Unable to process JSON: {}", exception.getMessage());
+    LOGGER.debug("Full JsonProcessingException", exception);
```

Comment:
omega nit:

The other message uses lower case (`runtimeException`) while this uses upper (`JsonProcessingException`). Also, I imagine the type is part of the `Exception.toString`